python/verticox/vantage6.py

```diff
@@ -0,0 +1,145 @@
+import time
+from typing import List
+
+import grpc
+import pandas as pd
+from vantage6.client import ContainerClient
+from vantage6.tools.util import info
+
+from verticox import datanode
+from verticox.aggregator import Aggregator
+from verticox.grpc.datanode_pb2_grpc import DataNodeStub
+
+PORT = 8888
+MAX_RETRIES = 20
+GRPC_OPTIONS = [('wait_for_ready', True)]
+SLEEP = 5
+DATANODE_TIMEOUT = 60 * 60
+DATA_LIMIT = 10
```

Comment:
Will these soon be set somewhere else? If not, maybe some brief comments could help remind what these actually control